ocs_ci/ocs/scale_noobaa_lib.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import time
+
+from ocs_ci.helpers import helpers
+from ocs_ci.utility import templating
+from ocs_ci.ocs import constants
+
+from ocs_ci.ocs.utils import oc_get_all_obc_names
+from ocs_ci.utility.utils import run_cmd
+
+log = logging.getLogger(__name__)
+
+
+def construct_obc_creation_yaml_bulk_for_kube_job(no_of_obc, sc_name, namespace):
+    """
+    Constructing obc.yaml file to create bulk of obc's using kube_job
+
+    Args:
+        no_of_obc(int): Bulk obc count
+        sc_name (str): storage class name using for obc creation
+        namespace(str): Namespace uses to create bulk of obc
+    Returns:
+         obc_dict_list (list): List of all obc.yaml dicts
+
+    """
+
+    # Construct obc.yaml for the no_of_required_pvc count
+    # append all the obc.yaml dict to obc_dict_list and return the list
+    obc_dict_list = list()
+    for i in range(no_of_obc):
+        obc_name = helpers.create_unique_resource_name("test", "obc")
+        obc_data = templating.load_yaml(constants.MCG_OBC_YAML)
+        obc_data["metadata"]["name"] = obc_name
+        obc_data["metadata"]["namespace"] = namespace
+        obc_data["spec"]["storageClassName"] = sc_name
+        obc_data["spec"]["bucketName"] = obc_name
+        obc_dict_list.append(obc_data)
+
+    return obc_dict_list
+
+
+def check_all_obc_reached_bound_state_in_kube_job(
+    kube_job_obj, namespace, no_of_obc, timeout=300
+):
+    """
+    Function to check either bulk created OBCs reached Bound state using kube_job
+
+    Args:
+        kube_job_obj (obj): Kube Job Object
+        namespace (str): Namespace of OBC's created
+        no_of_obc (int): Bulk OBC count
+        timeout: a timeout for all the obc in kube job to reach bound status
```

Comment:
I think it's important to note the timeout is in seconds and will be checked 20 times.
Alternatively, I think it'd be best to replace the hardcoded `*20` with a parameter given by the user

Reply:
Added "wait_time" parameter and updated the timeout in seconds.